models/datasets/reservoir/ethereum/tokens_beta.sql

```diff
@@ -0,0 +1,41 @@
+{{ config(
+    tags = ['dunesql'],
+    schema = 'reservoir',
+    alias = alias('tokens_beta'),
+    post_hook = '{{ expose_dataset(\'["ethereum"]\',
+                \'[""]\') }}'
+    )
+}}
+select
+    t.created_at,
+    t.collection_id,
+    _unsafe_0x_or_base58_to_varbinary(t.contract) as contract,
+    _unsafe_0x_or_base58_to_varbinary(t.floor_ask_maker) as floor_ask_maker,
+    _unsafe_0x_or_base58_to_varbinary(t.floor_ask_id) as floor_ask_id,
+    _unsafe_0x_or_base58_to_varbinary(t.owner) as owner,
```

Comment:
we haven't been using these on dunesql spells. this means the third party data is uploaded on spark, then needs converted to dunesql datatypes?

i think the approach has been `fron_base58()` function (or something like that, off top of head) to convert to varbinary

Reply:
@dsalv is it safe to use some other function here?

Reply:
It should be fine, but checking using the raw data is better. I can do it tomorrow or you can also check in the explorer

Reply:
>but checking using the raw data is better.

Could you elaborate on this I didn't understand it.

Reply:
` from_hex(t.owner)` should work. But it's better to run queries on raw tables and validate that the resulting data is same